File: pharmacie/src/main/java/io/pharmacie/models/Commune.java
package io.pharmacie.models;

public class Commune {
    private String nomCommune;
    private int codeWilaya;

    public int getWiyalaCode(){
        return this.codeWilaya;
    }
    public String toString() {
        return nomCommune;
    }

    public Commune(String name,int wilayacode){
        this.nomCommune = name;
        this.codeWilaya = wilayacode;
    }

    public String getNomCommune() {
        return nomCommune;
    }

    public int getCodeWilaya() {
        return codeWilaya;
    }

    public void setNomCommune(String nomCommune) {
        this.nomCommune = nomCommune;
    }

    public void setCodeWilaya(int codeWilaya) {
        this.codeWilaya = codeWilaya;
    }
}

File: pharmacie/src/main/java/io/pharmacie/models/Commune.kt
package io.pharmacie.models

class Commune(var nomCommune: String?, wilayacode: Int) {
    var wiyalaCode: Int = 0
        private set



    init {
        this.wiyalaCode = wilayacode
    }

    fun getCodeWilaya(): Int {
        return wiyalaCode
    }

    fun setCodeWilaya(codeWilaya: Int) {
        this.wiyalaCode = codeWilaya
    }
}
